target/linux/ath25/image/Makefile

```diff
@@ -73,32 +73,37 @@ endef
 TARGET_DEVICES += generic
```

Comment:
@aparcar How to deal with a generic device like that, which is nevertheless added to TARGET_DEVICES?

Reply:
I don't know how the `TARGET_DEVICES` works, can you elaborate?

Reply:
If I understand it correctly, it just means that "generic" is built as an actual device and not just a node with common definitions. So one would have to decide whether it is left with DEVICE_TITLE as a special case or one wants to introduce DEVICE_VENDOR etc. here too.

Reply:
My initial idea was to keep DEVICE_TITLE for such special cases, so I'd be fine if these special cases lack vendor etc.

Reply:
Okay, so keep it as it is.

Reply:
Ok, I will leave it as it is then.